src/sharding/ShardingManager.js

```diff
@@ -19,8 +19,9 @@ const Util = require('../util/Util');
  */
 class ShardingManager extends EventEmitter {
   /**
-   * The mode to spawn shards with for a {@link ShardingManager}. Either "process" to use child processes, or
-   * "worker" to use [Worker threads](https://nodejs.org/api/worker_threads.html).
+   * The mode to spawn shards with for a {@link ShardingManager}. Can be either one of:
+   * * 'process' to use child processes
+   * * 'worker' to use [Worker threads](https://nodejs.org/api/worker_threads.html)
    * @typedef {string} ShardingManagerMode
```

Comment:
It is but this isn’t done anywhere else on the library because if more options were added this would become huge and it also just repeats the information above so I don’t think it’s necessary

Reply:
There won't be any more.

Reply:
Yeah for this specific case I know there won’t but for the sake of consistency I still think it should say string + we’d be repeating information that was mentioned above so I personally don’t see a point